packages/blockchain/bch/src/lib/utils/bch.address.ts

```diff
@@ -4,6 +4,8 @@ import coininfo from 'coininfo'
 import bcash from '@tatumio/bitcoincashjs2-lib'
 import { networks } from 'bitcoinjs-lib'
 
+var Buffer = require('safe-buffer').Buffer
```

Comment:
afaik ts lint should force to use import instead of require. anyway please change it to import and const :)